examples.md

```diff
@@ -550,3 +551,17 @@ whenever possible:
     restore-keys: |
       ${{ runner.os }}-spm-
 ```
+
+## * - Bazel
+
+Bazel cache has a good handle to check if cached content should be rebuild or not based on its inputs like a hash(command + files). So using the latest cache of the branch is enough, no need to suffix with `hashFiles('**/...')`.
+
+```yaml
+- name: Cache Bazel
+  uses: actions/cache@v2
+  with:
+    path: |
+      ~/.cache/bazelisk
```

Comment:
Caching Bazelisk artifacts counts against the quota, so it's not a great choice for larger repos.

Reply:
@PiotrSikora I've explored this a bit further. The size increase is negligible, comparatively... because with Bazel, even for a relatively simple e.g. Java Bazel project (but with https://rules-proto-grpc.com/en/latest/example.html#step-3-write-a-build-file; which pulls in a shitload of C++ building, [because of this](https://github.com/rules-proto-grpc/rules_proto_grpc/issues/165)), one GitHub Actions Cache is 920 MB for me - and `~/.cache/bazelisk/` (filled with `bazel-6.0.0-linux-x86_64`) only adds another 51M on top of that. With GitHub now [providing 10 GB total size of all caches in a repository](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy), adding this seems like a worthwhile trade-off (to me). Even if it doesn't noticeable decrease build speed (at least for me), it still seems "nicer" and perhaps also more reliable not to see a `Downloading https://releases.bazel.build/6.0.0/release/bazel-6.0.0-linux-x86_64...` in EVERY build.

Reply:
True, but it's 50MB per hash, not total, so if your cache is otherwise 500MB, then it means that you're losing two slots in the cache.